README.md

```diff
@@ -200,6 +201,14 @@ import { ApolloClientResolversToken } from "fusion-apollo-universal-client";
 
 Optional - Provides the resolvers for [local state management](https://www.apollographql.com/docs/react/essentials/local-state.html).
 
+##### `ApolloBodyParserConfigToken`
+
+```js
+import { ApolloBodyParserConfigToken } from "fusion-apollo-universal-client";
```

Comment:
It would be super useful for people if we had a short code snippet in the readme on usage I think.